packages/core/src/lib/types/WebManifest.ts

```diff
@@ -32,6 +32,19 @@ export interface WebManifestShortcutJson {
 
 type WebManifestDisplayMode = 'browser' | 'minimal-ui' | 'standalone' | 'fullscreen';
 
+export interface ShareTargetParams {
```

Comment:
It may be worth adding a comment that these two data structures follow the definitions from here:

https://w3c.github.io/web-share-target/